pkgs/applications/misc/bazecor/default.nix

```
@@ -17,12 +17,15 @@ appimageTools.wrapAppImage rec {
     # Workaround for https://github.com/Dygmalab/Bazecor/issues/370
     postExtract = ''
       substituteInPlace \
-        $out/usr/lib/bazecor/resources/app/.webpack/main/index.js \
-        --replace \
+        $out/usr/lib/${pname}/resources/app/.webpack/main/index.js \
```

```suggestion
        $out/usr/lib/bazecor/resources/app/.webpack/main/index.js \
```

Don't reference `pname` everywhere. It's of no use.

---

> Don't reference `pname` everywhere. It's of no use.

This is a bit overzealous and opinionated for no valid reason other than personal preference as far as I can tell. 
I'm happy to adjust the PR to fit the contributing guidelines, but I don't think this is discouraged.

I can find many example in `nixpkgs` where exactly this approach is used:

https://github.com/NixOS/nixpkgs/blob/33d1e753c82ffc557b4a585c77de43d4c922ebb5/pkgs/os-specific/linux/uhk-agent/default.nix#L14-L19
https://github.com/NixOS/nixpkgs/blob/33d1e753c82ffc557b4a585c77de43d4c922ebb5/pkgs/by-name/co/codux/package.nix#L18-L25
https://github.com/NixOS/nixpkgs/blob/33d1e753c82ffc557b4a585c77de43d4c922ebb5/pkgs/tools/networking/requestly/default.nix#L17-L23

All of this to say, it's just a matter of where I took inspiration from and what you personally prefer and I don't feel like this debate is of any value.

---

Okay, I feel sorry about that.

---

That's ok, no hard feelings! ;)

---

This is raised in this issue: https://github.com/NixOS/nixpkgs/issues/277994

---

> This is raised in this issue: #277994

Which has been opened months ago without any new updates since January. Like I stated earlier, I think this is a waste of time and energy and this whole debate is quite pointless.

I will update the PR according to #277994 because I feel like we wasted entirely too much time and energy over something so irrelevant.